### PR TITLE
Admin dashboard

### PR DESCRIPTION
**Pull request by akash-011**
#### What does this PR do?
- Add UserFilter component
- Add AvailableUser component
- Add routes and restrictions for non-admins

#### Description of Task to be completed?
Add New Admin Dashboard designs and functionality
#### Where should the reviewer start?
File changes in the PR
#### How should this be manually tested?
Run the app , `yarn start` , and access the dashboard at `localhost/login` if you are designated as an admin

#### What are the relevant pivotal tracker stories?
[ #159198507]()

### CODE REVIEW

File: src/Components/AvailableUser/AvailableUser.Component.jsx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+          <SelectField
+            underlineStyle={{ display: 'none' }}
+            iconStyle={{ display: 'none' }}
+            labelStyle={{
+              textAlign: 'center',
+              marginRight: '1vw',
+              paddingRight: '0',
+              color: '#49abb0',
+              fontFamily: 'DIN Pro Extra Light',
+              fontWeight: 'normal',
```
Comment (kimobrian) [inline src/Components/AvailableUser/AvailableUser.Component.jsx]: Can we put these styles as variables then use them here. Doesn't look very good here.

File: src/actions/timelineAction.jsx [hunk 2]
```diff
@@ -66,7 +66,7 @@ export const addNote = (noteText, incidentId) => {
     return axios
       .post(notesUrl, {
         note: noteText,
-        userEmail: localStorage.getItem('email')
+        userId: localStorage.getItem('email')
```
Comment (kimobrian) [inline src/actions/timelineAction.jsx]: Why this change?

Comment (peterpaints) [reply src/actions/timelineAction.jsx]: There was a bug. The mockController/middleware was expecting `userId`, but the action was sending it `userEmail`

File: src/Components/AvailableUser/AvailableUser.Component.jsx (hunk 1 above)
Comment (peterpaints) [reply src/Components/AvailableUser/AvailableUser.Component.jsx]: Done.

File: src/actions/timelineAction.jsx (hunk 2 above)
Comment (kimobrian) [reply src/actions/timelineAction.jsx]: @peterpaints Has this been tested with the actual backend?

Comment (peterpaints) [reply src/actions/timelineAction.jsx]: I've removed the change. I've seen the error of my ways.
